th/2017/winter.json

```diff
@@ -0,0 +1,1283 @@
+{
+   "name":"ชื่อ",
+   "group":"แฟนซับ",
+   "message":"ไม่ทราบข้อมูล",
+   "lbl_tv":"TV",
+   "lbl_ova":"OVA/ONA/SP",
+   "lbl_movie":"Movie",
+   "tv":[
+      {
+         "name":"ACCA 13-ku Kansatsu-ka",
+         "image":"https://cdn.anilist.co/img/dir/anime/reg/21823-rO1rsx6OANhJ.jpg",
+         "group":[
+            {
+               "status":"release",
+               "detail":[
+                  {
+                     "name":"Shiniji-FS"
+                  },
+                  {
+                     "name":"VirgoNeo-FS"
+                  }
+               ]
+            },
+            {
+               "status":"release",
+               "detail":[
+                  {
+                     "name":"CrimsonV-FS"
+                  }
+               ]
+            }
+         ]
+      },
+      {
+         "name":"Akiba's Trip: The Animation",
+         "image":"https://cdn.anilist.co/img/dir/anime/reg/97636-RJRUVDqutPnX.jpg",
+         "group":[
+            {
+               "status":"release",
+               "detail":[
+                  {
+                     "name":"Seikai-Fansub"
+                  },
+                  {
+                     "name":"VirgoNeo-FS"
+                  }
+               ]
+            },
+            {
+               "status":"release",
+               "detail":[
+                  {
+                     "name":"KataFansub"
+                  },
+                  {
+                     "name":"KamiAmagumoSub"
```

Comment:
Can you sort the fansub by name, eg for this line:
```
{
	"detail": [{
		"name": "KamiAmagumoSub"
	}, {
		"name": "KataFansub"
	}]
}
```
because Ka**m**i (m) is before Ka**t**a (t)